.github/workflows/close-jira-issue.yml

```
@@ -20,22 +19,90 @@ jobs:
       - name: Extract Jira issue key from GitHub issue title
         id: extract-key
         run: |
```

actionlint (1.7.8)

"github.event.issue.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)